front/style/layout/_base.scss

```diff
@@ -26,6 +26,22 @@ svg polygon {
     border: none;
 }
 
+button {
+    &.link--deletered {
```

Comment:
maybe it'll be nice to have a more generic class name to transform one button to a link and in `_button.scss` file?
ex:
```css
@mixin btn--as-link {
  /* your css here */
}

button {
  & .btn--as-link {
    @include btn--as-link; 
  }
}

input {
  &[type="submit"],
  &[type="button"],
  &[type="reset"] {
    & .btn--as-link {
      @include btn--as-link;
    }
  }
}
```